ansible/modules/hashivault/hashivault_oidc_auth_role.py

```diff
@@ -185,16 +185,19 @@ def hashivault_oidc_auth_role(module):
     if not desired_state['token_policies'] and desired_state['policies']:
         desired_state['token_policies'] = desired_state['policies']
     desired_state.pop('policies', None)
+    desired_state['path'] = mount_point
 
     changed = False
     current_state = {}
     try:
-        current_state = getattr(client.auth, mount_point).read_role(name=name)['data']
+        current_state = getattr(client.auth, 'oidc').read_role(name=name, path=mount_point)['data']
```

Comment:
This should probably be `client.auth.oidc.read_role(...`

Reply:
Thanks for your review. I fixed it.